cassandra-executor/src/test/java/io/mesosphere/mesos/frameworks/cassandra/MockExecutorDriver.java

```diff
@@ -0,0 +1,123 @@
+package io.mesosphere.mesos.frameworks.cassandra;
+
+import com.google.protobuf.InvalidProtocolBufferException;
+import org.apache.mesos.Executor;
+import org.apache.mesos.ExecutorDriver;
+import org.apache.mesos.Protos;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+public class MockExecutorDriver implements ExecutorDriver {
```

Comment:
What's the motivation for the mock implementations rather than using something like mockito?